### ISSUE TITLE
Including accessibility module overrides legend.itemStyle

### ISSUE THREAD

**Issue by kluvin** [highcharts/highcharts#18620]
Removing the accessibility module consistently resolves the issue.

#### Expected behaviour
Hovering legend shows no change of cursor

#### Actual behaviour
Hovering legend shows change of cursor

#### Live demo with steps to reproduce
https://jsfiddle.net/kluvin/nc72sfku/2/

#### Product version
Introduced in 7.1.0, included in later

#### Affected browser(s)
Firefox, Chrome

**Comment by pawellysy** [comment]
Hello @kluvin, In my opinion, this is a desired change. Since this element is clickable, then the cursor should change, when hovering over the legend point as a default behavior. You can manually disable this behavior yourself, the same way you are doing with the click event: https://jsfiddle.net/BlackLabel/7xc0hLyz/1/

Here is a forum post about this subject: https://www.highcharts.com/forum/viewtopic.php?t=41480
closing as `not a bug`.

**Comment by kluvin** [comment]
Maybe I left out some details. My thinking is that the behavior should not change depending on whether the accessibility module is included. The provided example still exhibits the same issue, and it doesn't matter if we disable the click event, or override the cursor.

The forum post about changing the CSS does not seem related to this issue. It should not be necessary to override styles applied by accessibility.js.

I find that a wontfix label is more appropriate than a `not a bug` in this case.

**Comment by oysteinmoseng** [comment]
Internal note: Send in the option to the proxy button styling.
